src/frequenz/channels/merge.py

```diff
@@ -14,16 +14,16 @@
 class Merge(Receiver[T]):
     """Merge messages coming from multiple channels into a single stream.
 
-    For example, if there are two channel receivers with the same type, they
-    can be awaited together, and their results merged into a single stream, by
-    using `Merge` like this:
+    If there are two channel receivers with the same type, they can be awaited
+    together, and their results merged into a single stream.
```

Comment:
This is a specialization of the first line in the docstring,  and is suitable only for the example,  not as a general description.  I think it makes sense to move this paragraph under the `Example` section as well.

Reply:
I left it as before but inside the example.